manifests/crds/argoproj.io_applicationset.yaml

```diff
@@ -0,0 +1,6414 @@
+
+---
```

Comment:
Think this file can be removed? In this version of the file, the plural is `applicationset` rather than `applicationsets` for some reason :thinking:.